pyproject.toml

```diff
@@ -13,34 +13,21 @@ include = ["data/mixnorm", 'data/mcfa']
 
 [tool.poetry.dependencies]
 python = ">=3.8"
-sphinx = {version = "^4", optional = true}
-sphinx-redactor-theme = {version = "^0.0.1", optional = true}
-sphinx-hoverxref = {version = "*", optional = true}
-jinja2 = {version = "<3.1", optional = true}
 aiohttp = ">=3.8"
+click = ">=8.1.2"
+importlib-resources = ">=5.10.2"
+lmfit = ">=1.2.0"
 mcfa = "^0.1"
-pyqtgraph='>=0.13'
-pyqt5='>=5.15.9'
-pandas = "^1.4.2"
 numpy = ">=1.22.3"
-click = ">=8.1.2"
+pandas = ">=1.4.2"
+pyqtgraph='>=0.13'
 rich = ">=12.2.0"
-sphinx_design = "^0.3.0"
-sphinx-copybutton = "^0.5.0"
-furo = "^2022.9.15"
-scikit-learn = "^1.2.1"
+scikit-learn = ">=1.2.1"
 space-rocks = ">=1.7.2"
-tox = "^4.4.5"
-importlib-resources = "^5.10.2"
-lmfit = "^1.2.0"
 
 [tool.poetry.extras]
-docs = ["sphinx", "sphinx-redactor-theme"]
-
-[tool.poetry.dev-dependencies]
-sphinx-redactor-theme = "^0.0.1"
-sphinx-hoverxref = {version = "*", optional = true}
-jinja2 = "<3.1"
+docs = ["sphinx", "sphinx-redactor-theme", "sphinx_design", "sphinx-hoverxref", "jinja2", 'furo', "sphinx-copybutton"]
+gui = ['pyqt5']
```

Comment:
I think `pyside6` would have out-of-the-box binary compatibility with the most platforms.